src/TootleLib/clustering.cpp

```diff
@@ -202,8 +202,12 @@ static int MoveFaces(Mesh& mesh, std::vector<int>& seeds, std::vector<int>& clus
     }
 
     fMaxDist = cost[f];
+    
+#ifdef _DX11_1_
+    debugf("Max: %.6f, Avg: %.6f", fMaxDist, fAvgDist);
+#else
     debugf(("Max: %.6f, Avg: %.6f", fMaxDist, fAvgDist));
-
+#endif
```

Comment:
Could you use the original debugf macro?  The same review comment applies for the same changes in other files.